docs/Developer Documentation/Logger/index.md

```diff
@@ -0,0 +1,159 @@
+# SDL Logger
+
+Open source version of SDL is using log4cxx for logging.
+Because of lack of log4cxx stability OEM should have an ability to replace log4cxx logger with any other logging boost or syslog.
+
+
+SDL implements Logger abstration for easily replacement of logging library.
+
+|||
+High Level Design
+![Logger](./assets/high_level_design.png)
+|||
+
+
+All components have aceess to `Logger` interface and use it for logging.
```

Comment:
Please change to:
All components have access to the `Logger` interface and use it for logging.

Reply:
Fixed with  
aeadebe